_posts/en/newsletters/2023-02-15-newsletter.md

```diff
@@ -0,0 +1,259 @@
+---
+title: 'Bitcoin Optech Newsletter #238'
+permalink: /en/newsletters/2023/02/15/
+name: 2023-02-15-newsletter
+slug: 2023-02-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Continued discussion about block chain data storage:** several
+  threads on the Bitcoin-Dev mailing list this week saw continued
+  discussion about storing data in the block chain.
+
+    - *Offchain coin coloring:* Anthony Towns [posted][towns color] a
+      summary of a protocol currently being used for assigning special
+      meaning to certain transaction outputs, a class of techniques
+      generally called *coin coloring*.  He also summarized a related
+      protocol used for storing encoded binary data in Bitcoin
+      transactions and associating it with particular colored coins.
+      After summarizing the current state of affairs, he described a
+      method for storing data using the [nostr][] message transfer
+      protocol and associating it with colored coins that could be
+      transferred in Bitcoin transactions.  This would have several
+      advantages:
+
+      - *Reduced costs:* no transaction fees need to be paid for data
+        stored offchain.
+
+      - *Private:* two people can exchange a colored coin without
+        anyone else knowing anything about the data it references.
+
+      - *No transaction required for creation:* data can be associated
+        with an existing UTXO; there's no need to create a new UTXO.
+
+      - *Censorship resistant:* if the association between the data
+        and the colored coin is not widely known, then transfers of
+        the colored coin are just as censorship resistant as any other
+        onchain Bitcoin payment.
+
+      Considering the censorship resistant aspect, Towns argues that
```

Comment:
This might be the German coming out in me, but I would have put:
```suggestion
      Considering the censorship resistance aspect, Towns argues that
```